internal/app/server/server.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Envoyproxy Authors
```

Comment:
We don't need to have license text at the top of every file, so I would advise deleting it and not adding it to new files, just an FYI.

Reply:
Just saw this, was trying to be consistent with go-control-plane, but I can remove them :)